plugins/infinispan52/src/main/java/org/radargun/service/InfinispanHotrodService.java

```diff
@@ -71,7 +71,10 @@ public boolean isRunning() {
       return managerNoReturn != null && managerNoReturn.isStarted();
    }
 
-   protected TcpTransportFactory getTransportFactory(RemoteCacheManager manager) {
+   /*
+    * ISPN10 removed TcpTransportFactory and a class not found exception will throws.
+    */
+   protected Object getTransportFactory(RemoteCacheManager manager) {
```

Comment:
Why is the return type `Object` here?

Reply:
ISPN10 removed TcpTransportFactory and a class not found exception will throws.